openstack_retriever/requirements.txt

```diff
@@ -0,0 +1,2 @@
+lightspeed-rag-content @ git+https://github.com/road-core/rag-content@main
```

Comment:
ideally we would use pyproject.toml and sync pdm, like we do for other tools but this can be done afterwards.